app/src/main/java/agency/five/codebase/android/movieapp/ui/component/MovieCard.kt

```diff
@@ -0,0 +1,52 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.R
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+
+
+@Composable
+fun MovieCard(
+    imageUrl: String,
+    modifier: Modifier = Modifier,
+) {
+    Card(
+        modifier = modifier.size(122.dp, 179.dp),
```

Comment:
This does not look flexible enough. You have to think more in a Compose way. 
By setting the size you have limited your function a lot. Simply, provide a modifier, then later everyone can make their own version with their own sizing.